llvm/lib/CodeGen/GlobalISel/CallLowering.cpp

```diff
@@ -167,8 +161,9 @@ bool CallLowering::lowerCall(MachineIRBuilder &MIRBuilder, const CallBase &CB,
     // callee must be in the same TU and therefore we can direct-call it without
     // worrying about it being out of range.
     Info.Callee = MachineOperand::CreateGA(cast<GlobalValue>(CalleeV), 0);
-  } else
+  } else {
```

Comment:
Are the formatting/braces changes in this commit related to the PR?

Reply:
yes they are, my bad.

Reply:
The formatting of that code is incorrect because "if one of the if blocks has braces, all in the chain must have". I don't know why did I fix that in this PR I did not pay enough attention :). Maybe I should undo this, because it pollutes the git history.

Reply:
I will squash all the commits and remove stuff that pollutes the history like this.